resources/lang/nl/messages.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+return [
+    'activate_account_notification_body' => 'Je ontvangt deze e-mail, omdat we een verzoek ontvangen hebben om je wachtwoord te resetten.',
+    'activate_account_notification_subject' => 'Activeer je account',
+    'asset_container_blueprint_instructions' => 'Blueprints definiëren aanvullende aangepaste velden die te gebruiken zijn als je assets bewerkt.',
+    'asset_container_create_folder_instructions' => 'Indien ingeschakeld, is kunnen gebruikers mappen aanmaken in deze container.',
```

Comment:
Voorstel:
Indien ingeschakeld kunnen gebruikers mappen aanmaken in deze container.